backend/geonature/core/ref_geo/routes.py

```diff
@@ -169,18 +169,19 @@ def get_areas():
     """
     # change all args in a list of value
     params = {key: request.args.getlist(key) for key, value in request.args.items()}
-    if params['as_geojson'][0] == "true":
+    if params.get('as_geojson')[0] == "true":
```

Comment:
ça ne fonctionne pas si params['as_geojson'] est `None`.
Voir ma proposition de code précedente: 
` if params.get('as_geojson') and params.get('as_geojson')[0] == "true":`